samples/server/petstore/nodejs/api/swagger.yaml

```diff
@@ -356,8 +356,8 @@ paths:
         description: "ID of pet that needs to be fetched"
         required: true
         type: "integer"
-        maximum: 5
-        minimum: 1
+        maximum: 5.0
+        minimum: 1.0
```

Comment:
Created https://github.com/swagger-api/swagger-codegen/issues/4649 for tracking.